Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.8.1-onbuild
```

Comment:
If you build your binary in travis and just add that to the docker file then you can go `FROM scratch` which will give you a tiny container.